src/Guard.js

```diff
@@ -13,12 +13,12 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  notice_end */
- 
+
 import * as utils from './utils';
 
 export default class Guard {
-    static isDefined(value, message) {
-        if (typeof value === 'undefined') {
+    static isNotNullOrUndefined(value, message) {
+        if (typeof value === 'undefined' || value === null) {
```

Comment:
`if(value != null)` should do the trick?

https://codepen.io/anon/pen/pRjarN?editors=1111